python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -142,34 +143,33 @@ def run(self, exposure, templateExposure, subtractedExposure, psfMatchingKernel,
             Y-pixel coordinate to use for computing constant matching kernel to use
             If `None` (default), then use the center of the image.
         svar : `float`, optional
-            image variance for science image
+            Image variance for science image
             If `None` (default) then compute the variance over the entire input science image.
         tvar : `float`, optional
             Image variance for template image
             If `None` (default) then compute the variance over the entire input template image.
 
         Returns
         -------
-        result : `Struct`
-            a `lsst.pipe.base.Struct` containing:
-
+        result : `lsst.pipe.base.Struct`
```

Comment:
This is a result of dropping the _correction kernel_ from the return. I am not 100% sure that we don't want to return any additional information; I'll postpone the removal of the Struct for the spatially varying correction implementation DM-23857. 

Considerations: It'd be logical to return something about the correction, but... 

-  returning the Fourier space correction needs the freqSpaceShape and may not suit for any outside operation.
- Returning the image space correction (full image size) needs one additional, otherwise unnecessary, IFT. Also where the origin is placed would need detailed description or additional, otherwise unnecessary, copying (re-centering) of the array. 
- If a matchingKernel size cropped version is returned then we throw away the main lesson from this whole change i.e. that the correction can be larger than the original matching kernel in general so should not be used.